server.js

```diff
@@ -2989,6 +2989,39 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+camp.route(/^\/sourcegraph\/([\s\S]+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+
+  var repo = match[1];  // eg, `prismic` or `foo/bar`.
+  var format = match[2];
+  var apiUrl = "https://sourcegraph.com/.api/repos/"+repo+"/-/shield"
+  var badgeData = getBadgeData('build', data);
```

Comment:
Are you sure that "build" is the best left-hand-side description of what the badge does? I feel like "dependents" or "used by" would be more appropriate, if I understand this badge correctly.